compiler/src/Language/Bond/Codegen/Java/Class_java.hs

```diff
@@ -301,6 +301,18 @@ makeStructBondTypeMember_deserializeStructFields declName declParams structField
                 verifyField Field {..} = [lt|this.#{fieldName}.verifyDeserialized(__has_#{fieldName});|]
 
 
+-- given struct class name, generic type parameters, and struct fields, builds text for implementation of
+-- the StructBondTypeImpl.deserializeStructFields method for UntaggedProtocolReaders
+makeStructBondTypeMember_deserializeStructFields_untagged :: String -> [TypeParam] -> [Field] -> Text
+makeStructBondTypeMember_deserializeStructFields_untagged declName declParams structFields = [lt|
+        @Override
+        protected final void deserializeStructFields(#{methodParamDecl}) throws java.io.IOException {#{newlineBeginSep 3 serializeField structFields}
+        }|]
+            where
+                methodParamDecl = [lt|com.microsoft.bond.BondType.UntaggedDeserializationContext context, #{typeNameWithParams declName declParams} value|]
+                serializeField Field {..} = [lt|this.#{fieldName}.deserialize(context);|]
```

Comment:
>this.#{fieldName} [](start = 48, length = 17)

Need to assign the result to a field in the value:

value.<field> = this.<field>.deserialize(context)